### PR TITLE
docs(oidc): add docs for 'resolve_distributed_claims'

### PR DESCRIPTION
**Pull request by jschmid1**
### Summary
https://openid.net/specs/openid-connect-core-1_0.html#DistributedExample

### Reason
Distributed/Aggregated Claims are defined in the openid-connect spec.
Internal: FTI-2978

### Testing
Testing is included in the respective plugin/lib implementations.

Signed-off-by: Joshua Schmid <jaiks@posteo.de>

dependent on:

* https://github.com/Kong/kong-ee/pull/2856
* Kong/kong-openid-connect#38

### CODE REVIEW

**Comment by lena-larionova** [comment]
@jschmid1 this work is for 2.8, correct? If so, can you please retarget this PR to the `release/gateway-2.8` branch?

**Comment by jschmid1** [comment]
> @jschmid1 this work is for 2.8, correct? If so, can you please retarget this PR to the `release/gateway-2.8` branch?

@lena-larionova thanks, done!

File: app/_hub/kong-inc/openid-connect/index.md [hunk 1]
```diff
@@ -1496,6 +1496,15 @@ params:
       default: true
       datatype: boolean
       description: Cache the user info requests.
+    - name: resolve_distributed_claims
+      required: false
+      default: false
+      datatype: boolean
+      description: |
+        Distributed Claims are represented by using special _claim_names and _claim_sources members of the JSON object containing the Claims.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/openid-connect/index.md:1504]:
```suggestion
        Distributed claims are represented by the `_claim_names` and `_claim_sources` members 
        of the JSON object containing the claims.
```

File: app/_hub/kong-inc/openid-connect/index.md [hunk 2]
```diff
@@ -1496,6 +1496,15 @@ params:
...
+        This config option allows explicitly enables resolving these distributed claims.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/openid-connect/index.md:1505]:
```suggestion
        If this parameter is set to `true`, the plugin explicitly resolves these distributed claims.
```

**Review by kikito — APPROVED** [review]

**Pull request merged**